app/models/table_preference.rb

```diff
@@ -0,0 +1,10 @@
+class TablePreference < ApplicationRecord
```

Comment:
do we care about auditing or search?

Reply:
I don't think so, this is just UI preferences, not anything that has impact on the system. It's similar to the dashboard widget settings which we don't audit or search.